apps/src/blockly/customBlocks/googleBlockly/mutators/procedureDefMutator.js

```diff
@@ -81,18 +82,25 @@ export const procedureDefMutator = {
         this.description = node.textContent;
       }
     }
+
+    this.userCreated = readBooleanAttribute(
+      xmlElement,
+      'userCreated',
+      FALSEY_DEFAULT
+    );
     this.setStatements_(xmlElement.getAttribute('statements') !== 'false');
   },
 
   /**
-   * Returns the state of this block as a JSON serializable object.
-   * @returns The state of this block, eg the parameters and statements.
+   * Returns a JSON serializable value which represents the extra state of the block.
```

Comment:
I referenced the way these comments were written in the Blockly documentation a couple of times because the language was useful, so I proposed tweaking them here: https://developers.google.com/blockly/guides/create-custom-blocks/extensions